core/store/src/adapter/flat_store.rs

```diff
@@ -279,6 +279,7 @@ impl<'a> FlatStoreUpdateAdapter<'a> {
 
     // helper
     fn remove_range_by_shard_uid(&mut self, shard_uid: ShardUId, col: DBCol) {
+        assert!(col != DBCol::State, "can't range delete State column");
```

Comment:
nit: should we add `use delete_shard_uid_prefixed_state instead?`

Reply:
Yes, I though maybe not mentioning `delete_shard_uid_prefixed_state` so that there is less probability of someone using it, and it is should only be used for cleanup.

Reply:
Actually, I already added a warning in comment to `delete_shard_uid_prefixed_state`, so I will apply your sugestion to mention `delete_shard_uid_prefixed_state` here.

Reply:
I understand your point, let's keep it like it is